### ISSUE TITLE
Help with Train With Dataset WebFace42M Error

### ISSUE THREAD

**Issue by Harry-zklcdc** [mk-minchul/CVLface#23]
When I Try to Train Adaface-iR50 with Dataset WebFace42M at 8*A100 SXM 80G, the loss-fn is not Reduce after 1 epoch, and The Eval Acc don't  Increase.

But when I train with sampling 1% dataset at WebFace42M, It is running normally.

## Config

```yaml

```

## Train Log

![QQ_1731397903966](https://github.com/user-attachments/assets/c2b01def-528a-4f60-88e2-d398ddc5cb87)
![QQ_1731398152632](https://github.com/user-attachments/assets/bba803de-663d-4fc6-973d-5034c5a9c0d1)

**Comment by wuxuedaifu** [comment]
how long it take to run each epoch? did you set the num_class correctly? or need more epoch runs to see loss decrease？

**Comment by Harry-zklcdc** [comment]
> how long it take to run each epoch? did you set the num_class correctly? or need more epoch runs to see loss decrease？

Is Chinese? 

> how long it take to run each epoch?

About one and a half hour for each epoch

> did you set the num_class correctly?

yes, set `num_calasses` to 2059906, and `num_images` to `42474557`

> need more epoch runs to see loss decrease？

I set it to 26 which is same as the paper.

**Comment by wuxuedaifu** [comment]
how about initializing the network with the pretrained models provided by the author？

**Comment by Harry-zklcdc** [comment]
> how about initializing the network with the pretrained models provided by the author？

I don't use the pretrain model.

**Comment by wensi2690** [comment]
hi, could you share the WebFace42M dataset with me , i can not find it in public website!
